### PR TITLE
add ManagedClusterSetBinding

### PR DESCRIPTION
**Pull request by haoqing0110**
Signed-off-by: haoqing0110 <qhao@redhat.com>

### CODE REVIEW

File: content/en/concepts/managedclusterset.md [hunk 1]
```diff
@@ -140,6 +148,26 @@ status:
     type: ClusterSetEmpty
 ```
 
+The `ManagedClusterSetBinding` can also be checked by the command
```
Comment (qiujian16) [inline content/en/concepts/managedclusterset.md:151]: maybe also a clusteradm example

Comment (haoqing0110) [reply content/en/concepts/managedclusterset.md:151]: @qiujian16 Hmm, do you mean `clusteradm get clustersets` example in the above lines? https://github.com/open-cluster-management-io/open-cluster-management-io.github.io/pull/314/files#diff-1b864eb069cba3ff069bca7032d56752097ffd160e6386f161ca9c806ae6354eL93

Comment (qiujian16) [reply content/en/concepts/managedclusterset.md:151]: yes, I think so

**Pull request merged**